INSTALL.md

```diff
@@ -239,6 +239,7 @@ docker run -p 9001:9000 --name clamav -d niilo/clamav-rest
 	{
 	    port: 3005
 	    dev: false
+		trustedProxy: 127.0.0.1/8 // This is optional, and default value is 'loopback'. if unnecesarry, you can remove this.
```

Comment:
We already include 'loopback' by default, which will trust '127.0.0.1/8' and '::1/128'.  Turning this into an empty array might be less confusing.

Reply:
removed value